tests/yarn_files/assignment.json

```diff
@@ -58,6 +58,26 @@
 		},
 		"colorID": 0
 	},
+	{
+		"title": "Null",
+		"tags": "Tag",
+		"body": "Test Line\n<<set $testvar = null>>\nTest Line After",
```

Comment:
can you explain a use case for this?

Reply:
This is actually in the Yarn Spinner documentation here: https://yarnspinner.dev/docs/writing/expressions-and-variables/

> Yarn Spinner also has a special value, called null. This value represents “no value”; any variable that hasn’t yet been assigned a value will be null.

I've found it necessary to initialize some variables with null values. Is there a reason not to support this?